src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -88,7 +89,7 @@ public Command parseCommand(String userInput) throws ParseException {
             return new ExitCommand();
 
         case LastContactCommand.COMMAND_WORD:
-            return new LastContactCommandParser().parse(arguments);
+            return new LastContactCommand(new IsLastContactedPredicate());
```

Comment:
I think since every LastContactCommand has the exact same IsLastContactedPredicated, we can perhaps explore just keeping a static variable is the contact that contains this predicate? 🤔 Like that we don't have to keep instantiating a new object every time the command is called. 👍